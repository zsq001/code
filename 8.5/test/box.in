5
4
-1000 1000 999 -7
1
1000
4
-1 100 4 -5
1
-4
1
0