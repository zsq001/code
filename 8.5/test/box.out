994
1000
92
-4
0
